packages/core/src/index.ts

```diff
@@ -283,6 +285,35 @@ Signal.prototype.subscribe = function (fn) {
 	});
 };
 
+Signal.prototype.set = function (value) {
```

Comment:
Any reason we don't reuse the implementation of the value setter here and do `this.value = value`?

Reply:
Good point :joy: